### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by odradek71**
update doc to match filenames

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ get('counter').then((val) =>
 );
 ```
 
-With the above, both `get` operations will complete first, each returning `undefined`, then each set operation will be setting `1`. You could fix the above by queuing the second `get` on the first `set`, but that isn't always each across multiple pieces of code. Instead:
+With the above, both `get` operations will complete first, each returning `undefined`, then each set operation will be setting `1`. You could fix the above by queuing the second `get` on the first `set`, but that isn't always feasible across multiple pieces of code. Instead:
```
Comment (odradek71) [inline README.md:148]: just guessing what was intended

Comment (jakearchibald) [reply README.md:148]: I guess I was going for "easy" but "feasible" works too.

**Comment by jakearchibald** [comment]
Thank you!

**Comment by odradek71** [comment]
Thank you for this tool, does just what I needed!  I'm happy to make a small contribution :-)

**Pull request merged**